code/further_analyses/ica_analysis.py

```diff
@@ -0,0 +1,103 @@
+"""
+A set of functions for extracting and visualizing results from applying independent components analysis (ICA) on the
+preprocessed neuroimaging data.
+"""
+
+import os
+import numpy as np
+import nibabel as nib
+import matplotlib.pyplot as plt
+
+def prep_data(data_path='/Users/nimahejazi/Dropbox/159_projData/preprocessed/',subj_num):
```

Comment:
If you review and merge my pull request, I've put this prepare_data function into general_utils. It might be better if you update your script to use that function so that we don't have duplicate code. Thanks!